### PR TITLE
feat(dapp-console): create app data model

### PR DESCRIPTION
**Pull request by tremarkley**
Closes https://github.com/ethereum-optimism/ecopod/issues/911

### CODE REVIEW

**Comment by tremarkley** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ecosystem/195?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#195** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ecosystem/195?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#194** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ecosystem/194?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#183** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ecosystem/183?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tremarkley and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dapp-console-api/src/models/apps.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+export const apps = pgTable('apps', {
+  createdAt: timestamp('created_at', { withTimezone: true })
+    .defaultNow()
+    .notNull(),
+  updatedAt: timestamp('updated_at', { withTimezone: true })
+    .defaultNow()
+    .notNull(),
+  id: uuid('id').defaultRandom().primaryKey(),
+  entityId: uuid('entity_id').references(() => entities.id),
```
Comment (nitaliano) [inline apps/dapp-console-api/src/models/apps.ts:18]: it might be good for this to be not null

File: apps/dapp-console-api/src/models/apps.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { integer, pgTable, timestamp, uuid, varchar } from 'drizzle-orm/pg-core'
+
+import { entities } from './entities'
+
+enum AppState {
+  ACTIVE = 'active',
+  DISABLED = 'disabled',
+}
+
+export const apps = pgTable('apps', {
```
Comment (nitaliano) [inline apps/dapp-console-api/src/models/apps.ts:10]: i don't believe postgres will create an index for foregin keys which is what the entity_id would be considered we might need to manually add an index for it

Comment (tremarkley) [reply apps/dapp-console-api/src/models/apps.ts]: good call, just added the index

File: apps/dapp-console-api/src/models/apps.ts (hunk 1 above)
Comment (tremarkley) [reply apps/dapp-console-api/src/models/apps.ts]: done

**Review by nitaliano — APPROVED** [review]

**Pull request merged**